package.json

```diff
@@ -60,27 +62,29 @@
   },
   "scripts": {
     "compile-styles": "lessc ./node_modules/antd/dist/antd.less ./src/styles/antd.css --js",
+    "dev": "yarn prestart && next dev",
     "prestart": "yarn compile-styles",
-    "start": "ESLINT_NO_DEV_ERRORS=true REACT_APP_VERSION=$(git rev-parse --short HEAD) react-scripts start",
-    "build": "REACT_APP_VERSION=$(git rev-parse --short HEAD) react-scripts build",
+    "start": "ESLINT_NO_DEV_ERRORS=true NEXT_PUBLIC_VERSION=$(git rev-parse --short HEAD) next start",
     "prebuild": "yarn compile-styles && yarn i18n:compile",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "build": "NEXT_PUBLIC_VERSION=$(git rev-parse --short HEAD) next build",
+    "test": "jest test",
     "lint": "eslint --ext .js,.jsx,.ts,.tsx ./",
     "lint:fix": "yarn lint --fix",
     "lint-staged": "lint-staged",
     "prepare": "husky install",
     "i18n:extract": "./scripts/extract-translations.sh",
-    "i18n:compile": "lingui compile",
+    "i18n:compile": "NODE_ENV=development lingui compile",
```

Comment:
**question** curious why this is needed.

Reply:
o i c https://github.com/lingui/js-lingui/issues/952#issuecomment-765449330

all g